test/passing/ocp_indent_options.ml.ref

```diff
@@ -2,7 +2,7 @@ let _ =
   let f x y =
     match x with
     | None -> false
-    | Some loooooooooooooooooooooooooooooooooooooooooooooooooooooooooong -> (
-      match y with Some _ -> true | None -> false )
+    | Some loooooooooooooooooooooooooooooooooooooooooooooooooooooooooong
+    -> ( match y with Some _ -> true | None -> false )
```

Comment:
I think that it's correct to break here (I haven't checked the margin carefully), but the indentation should be 2 spaces more. Seems to be the same as the case from selection.ml.